ArduPlane/Plane.cpp

```diff
@@ -514,6 +514,10 @@ void Plane::update_control_mode(void)
     update_fly_forward();
 
     control_mode->update();
+
+#if MODE_AUTOLAND_ENABLED
+    check_takeoff_direction();
```

Comment:
```suggestion
    mode_autoland.check_takeoff_direction();
```
move function into mode and then call via the refence.

Reply:
done